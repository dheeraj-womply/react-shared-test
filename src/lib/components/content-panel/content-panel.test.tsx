import React from 'react'
import { ContentPanel } from './index'
import renderer from 'react-test-renderer'

test('Error Panel DOM elements are OK', () => {
    const component = renderer.create(<ContentPanel title={'Womply'} >
        <p>Womply</p>
    </ContentPanel>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
