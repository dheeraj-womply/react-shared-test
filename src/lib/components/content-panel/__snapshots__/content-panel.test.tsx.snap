// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Panel DOM elements are OK 1`] = `
<div
  className="sc-AxjAm cNpNtW"
>
  <div
    className="sc-AxirZ byOHfS"
  >
    Womply
  </div>
  <div
    className="sc-AxiKw emfjN"
  >
    <p>
      Womply
    </p>
  </div>
</div>
`;
